README.md

```diff
@@ -1,7 +1,11 @@
 # ChainerUI
 
+![training_captures](https://user-images.githubusercontent.com/414255/34244148-637e6630-e667-11e7-8a6b-9a857fa806bc.gif)
```

Comment:
How do we update this image?

Reply:
I think this animation GIF is not official image (it is used for release blog), so it is better to prepare "official" sample images for document, and README links the URL path. Currently official document (like readthedocs)  is not published, and I uploaded the GIF on Github's user content.

Is it better to upload it on the repository path, like `docs/images`

Reply:
If we need to update the image before publish document, have to upload new image and rewrite the URL link path.